charts/system/kubelet-csr-approver/templates/common.yaml

```diff
@@ -1,5 +1,10 @@
 {{/* Make sure all variables are set properly */}}
 {{- include "tc.v1.common.loader.init" . }}
 
+{{- $configmapFile := include "kubelet-csr-approver.configmap.config" . | fromYaml -}}
+{{- if $configmapFile -}}
+  {{- $_ := set .Values.configmap "csr-config" $configmapFile -}}
+{{- end -}}
```

Comment:
```suggestion

```

As the envs need to be moved to the envs section of the container/workload config, these need to be yeeted as well.